ntbs-service/wwwroot/css/labResults.scss

```diff
@@ -56,4 +56,12 @@
   > .nhsuk-error-message {
     flex-basis: 100%;
   }
+}
+
+.unmatched-results-container {
+  background: #F0F4F5;
```

Comment:
As we're using this in a couple of places now it might be worth chucking this in colors.scss